docs/io/fits/performance.inc.rst

```diff
@@ -1,12 +1,19 @@
-.. note that if this is changed from the default approach of using an *include* 
+.. note that if this is changed from the default approach of using an *include*
    (in index.rst) to a separate performance page, the header needs to be changed
-   from === to ***, the filename extension needs to be changed from .inc.rst to 
+   from === to ***, the filename extension needs to be changed from .inc.rst to
    .rst, and a link needs to be added in the subpackage toctree
 
 .. _astropy-io-fits-performance:
 
 Performance Tips
 ================
 
-Here we provide some tips and tricks for how to optimize performance of code
-using `astropy.io.fits`.
+By default, :func:`astropy.io.fits.open` will open files using memory-mapping,
+which means that the data is not necessarily read into memory until it is
+needed. While memory-efficient, if memory is not a concern for you, you may
+find that you can get better performance by turning memory mapping off, which
+forces the data to be read into memory immediately:
```

Comment:
The gain of disabling memmap is not obvious to me. There is a discussion about this in the comparison with cfitsio (http://docs.astropy.org/en/latest/io/fits/appendix/faq.html#what-performance-differences-are-there-between-astropy-io-fits-and-fitsio). In theory it should be better to disable memmap when using the full array, but in practice I saw no difference when I tested this on data cubes (as far as I remember). 
So before advising to disable memap I think that we should have a more extensive comparison, to see if and when there is really a benefit.